### PR TITLE
Disable synced by default

### LINKED ISSUE #5141: Disable sync information in agent table by default
**Describe the solution you'd like**

The agent sync information is not relevant for everybody and we want it hidden in a default install. Furthermore, we want to delete the global sync indicator in the upper widget.

image

We want to change the texts:
IP -> IP address
OS -> Operating system

We need to ensure most common operating systems have their icons in the list.

Lastly, we want to re-distribute the space for each column, trying to avoid truncation.

Related issue https://github.com/wazuh/wazuh-kibana-app/issues/3402

### PR DESCRIPTION
**Pull request by asteriscos**
### Description
Hi team,

This PR: 

- Disables by default the `Synced` - `Registration Date` - `Last Keep Alive` fields of the Agents table and removes the metric on the top of the view. Even though it is disabled, it still can be enabled by checking the field in the table settings.
- It also changes the text `IP` and the text `OS` to `IP address` and to `Operating system` throughout the app. 
- Improves the Agents table dynamic layout
- Fixes Windows and MacOs operating system icons in the agents table

 
### Issues Resolved
Closes #5141 

### Evidence
image

### Synced field - Registration Date - Last Keep Alive

- Register agents in the manager
- Go to Agents overview
- Check the table does not include the `synced` field
- Check the top metrics don't include the `Synced` percentage
- Check the synced field can be enabled in the Agents table

#### Agents table 
image

#### Agents Inventory PDF report

- Go to an Agent inventory data
- Generate PDF report

image
image
image

#### Groups agents table

- Go to Management -> Groups
- Select a group
image

#### Agent overview

- Go to Agents
- Select an agent

image

#### Management cluster

- Go to Management -> Cluster
- Click on `Nodes` link

image
image

#### Office365 Panel

- Go to Office365 -> Panel
- Click on any Top Client IP Address table row to drill-down

image
image

#### Global Configuration Remote

- Go to Management -> Configuration
- Click on the Global configuration row
- Click on the Remote tab

image

#### Explore Agent modal

- Go to Security Events
- Click on Explore Agent button

image

### Redistribute Agents Table column widths automatically 

- Go to Agents Overview
- Add and remove all available columns to force different widths in the columns
- Check they adjust properly

### Verify all main agents operating system have the proper icon in the table

- Register Linux, Windows and MacOS agents
- Check they have the corresponding icon in the Agents Overview Table

>  **Important**
> To test it properly, delete the preview _**local storage**_ table configuration
>  image

## Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Tostti — CHANGES_REQUESTED** [review]
According to the [Issue](https://github.com/wazuh/wazuh-kibana-app/issues/5141), the `Registration date` and `Last keep alive` fields should also be disabled by default

File: public/components/management/cluster/node-list.tsx [hunk 1]
```diff
@@ -44,7 +44,7 @@ export const NodeList = withErrorBoundary (class NodeList extends Component {
             },
             {
                 field: 'ip',
-                name: 'IP',
+                name: 'IP Address',
```
Comment (Desvelao) [inline public/components/management/cluster/node-list.tsx:47]: **nitpick**: The other references use `IP address` with `address` in lowercase. Is there a reason to this case is different?

**Review by Desvelao — CHANGES_REQUESTED** [review]
**nitpick**: There are more files that contain references to IP and OS that we could be interested to change too.

Some of them:
- common/csv-key-equivalence.ts
- public/components/overview/office-panel/config/main-view-config.tsx
- public/components/visualize/visualizations.js (IP by Users)
- public/controllers/management/components/management/configuration/global-configuration/global-configuration-remote.js
- public/templates/management/management.html
- server/controllers/wazuh-reporting.ts
- server/integration-files/visualizations/overview/overview-hipaa.ts
- server/integration-files/visualizations/overview/overview-office.ts
- common/csv-key-equivalence.ts Review where it is affecting to
- common/compliance-requirements/pci-requirements.ts Not sure if this we could change it or should not changed.
- public/controllers/agent/components/agents-table.js
- public/controllers/management/components/management/configuration/vulnerabilities/vulnerabilities-providers.js
- public/controllers/overview/components/overview-actions/agents-selection-table.js
- server/lib/reporting/agent-configuration.ts
- server/lib/reporting/extended-information.ts
- public/controllers/management/components/management/groups/group-agents-table.js

Fix:
- public/components/management/cluster/node-list.tsx
- public/controllers/management/components/management/status/status-agent-info.js

**Comment by asteriscos** [comment]
> **nitpick**: There are more files that contain references to IP and OS that we could be interested to change too.
> 
> Some of them:
> 
> * common/csv-key-equivalence.ts
> * public/components/overview/office-panel/config/main-view-config.tsx
> * public/components/visualize/visualizations.js (IP by Users)
> * public/controllers/management/components/management/configuration/global-configuration/global-configuration-remote.js
> * public/templates/management/management.html
> * server/controllers/wazuh-reporting.ts
> * server/integration-files/visualizations/overview/overview-hipaa.ts

None found

> * server/integration-files/visualizations/overview/overview-office.ts

None found

> * common/csv-key-equivalence.ts Review where it is affecting to
> * common/compliance-requirements/pci-requirements.ts Not sure if this we could change it or should not changed.
> * public/controllers/agent/components/agents-table.js
> * public/controllers/management/components/management/configuration/vulnerabilities/vulnerabilities-providers.js

The space is too small and reducing "Operating system" to "OS" is worthwhile

> * public/controllers/overview/components/overview-actions/agents-selection-table.js
> * server/lib/reporting/agent-configuration.ts
> * server/lib/reporting/extended-information.ts

The "OS" in line 84 is justified to optimize the table space, as the manager name or agent name can be quite lengthy.

> * public/controllers/management/components/management/groups/group-agents-table.js
> 
> Fix:
> 
> * public/components/management/cluster/node-list.tsx
> * public/controllers/management/components/management/status/status-agent-info.js


Thanks for the suggestions.

**Review by Tostti — APPROVED** [review]
TEST:✔️
CR:✔️
PREVIOUS REVIEW CHANGES REQUESTED: ✔️

Evidence: 
imagen
imagen
imagen


LGTM

**Review by AlexRuiz7 — APPROVED** [review]
LGTM

**Comment by asteriscos** [comment]
## Table design 

The proposal to optimize the agents table is to make use of the **euiBasicTable** property` table-layout="auto"` and remove the fixed widths. Setting the table layout to auto make much better use of the blank spaced in columns in favor of lengthy column values. 

### side effect
The` table-layout="auto`" prop doesn't support truncating text when the table width is limited by the viewport. To avoid overflowing the table it wraps the lines of the values.

**Review by Tostti — APPROVED** [review]
imagen
imagen
imagen

imagen

TEST:✔️
CR:✔️

LGTM

**Comment by chantal-kelm** [comment]
**Replace OS to Operating system and IP to IP address**

**Agents table** 🟢 

<details>

**Chrome**  🟢 

image

**Firefox**  🟢 

image

**Safari**  🟢 

image
</details>

**Agents Inventory PDF report**

<details>

**Chrome**  🟢 

image

image

**Firefox**  🟢 

image

image

**Safari**  🟢 

image

image

</details>

**Groups agents table** 🟢 

<details>

**Chrome**  🟢 

image

**Firefox** 🟢 

image

**Safari** 🟢 

image
</details>

**Agent overview** 🟢 

<details>

**Chrome** 🟢 

image

**Firefox** 🟢 

image

**Safari** 🟢 

image

</details>

**Management cluster** 🟢 

<details>

**Chrome**  🟢 

image

image

**Firefox** 🟢 

image

image

**Safari** 🟢 

image

image

</details>

**Office365 Panel** 🟢 

<details>

**Chrome**  🟢 

image

image

**Firefox** 🟢 

image

image

**Safari** 🟢 

image

image

</details>

**Global Configuration Remote** 🟢 

<details>

**Chrome** 🟢 

image

**Firefox**  🟢 

image

**Safari**  🟢 

image

</details>

**Explore Agent modal** 🟢 

<details>

**Chrome**  🟢 

image

**Firefox**  🟢 

image

**Safari**  🟢 

image

</details>

**Redistribute Agents Table column widths automatically** 🔴 

<details>

**Chrome**  🔴 

image

**Firefox** 🔴 

image

**Safari** 🔴  

image

</details>

**Comment by asteriscos** [comment]
> 

Thanks for reporting this. I allowed the Registration date, IP address and version columns to wrap the line to prevent the table overflow.

image

**Comment by chantal-kelm** [comment]
> > 
> 
> Thanks for reporting this. I allowed the Registration date, IP address and version columns to wrap the line to prevent the table overflow.
> 
> image

Chrome

<details>

The table with all columns added looks good from 864px width onwards

image

</details>

Firefox

<details>
The table with all columns added looks good from 854px width onwards

image

</details>

Safari

<details>

The table with all columns added looks good from 854px width onwards

image

</details>

**Review by chantal-kelm — APPROVED** [review]
CR: ✅ 
TEST: ✅ 

Chrome

<details>

The table with all columns added looks good from 864px width onwards

image

</details>

Firefox

<details>
The table with all columns added looks good from 854px width onwards

image

</details>

Safari

<details>

The table with all columns added looks good from 854px width onwards

image

</details>

**Review by Tostti — APPROVED** [review]
Chrome

<details>
The table looks good in all the resolutions. When the is smaller than the width, it enables lateral scroll.

imagen

imagen

</details>

Firefox

<details>
The table looks good in all the resolutions. When the is smaller than the width, it enables lateral scroll.

imagen

imagen


</details>





TEST:✔️
CR:✔️

LGTM

**Review by Desvelao — APPROVED** [review]

**Pull request merged**